makers/staff_engineer.md

```diff
@@ -1,47 +1,78 @@
-* Develop and constantly refine technical, leadership, and team fit assessments (coding exercises, interview questions, etc) for interviews across the Engineering organization
-* Lead code/design/architecture reviews across teams and mentor & coach engineers to ensure work follows best practices & standards. Create and evolve brand wide practices and share with the rest of the organization.
-* Ensure that all brand decisions are in service of their customers
-* Help brand teammates communicate their design and architecture decisions throughout the company
-* Acts as a mentor to other senior and staff engineers who are developing their own mentees
-* Provide task definition and design/architecture direction for other engineers
-* Demonstrate expert knowledge of their discipline
-* Remain flexible & put customer needs first 
-* Accurately estimate time to complete projects given structure of team and deliver on those estimates
-* Allocate people within a team to maximise output of useful and team-goal-focused engineering work
-* Monitor performance of engineers on team, and partner with engineering managers on mentoring and growth
-* Look for ways to reduce distractions, interruptions, and unplanned work for themselves and others 
-* Challenge their team to demonstrate greater sense of urgency
-* Model bringing solutions to problems (not surfacing problems alone) and teach others on their team how to do the same
-* Be active in the technology community, positively contribute to Mariana Tek’s reputation, and find great candidates to grow the Mariana Tek team
-* Develop and constantly refine technical and team fit assessments (coding exercises, interview questions, etc) for interviews
-* Ensure code/design/architecture reviews happen on teams and that work follows established best practices and standards
-* Coordinate evaluation of tools and frameworks for the engineering team
-* Take responsibility for actions of their team, good and bad
+<!-- TODO: Add summary -->
+
+## Qualifications
+
+- Highly knowledgeable about our product, codebase, and tech stack.
+- Consistently delivers high-quality projects with cross-team impact.
+- Steps up to take leadership of a project or team when needed.
+- Respected by engineers across the company as a resource for technical knowledge, leadership, and mentoring.
+- Trusted by company leaders to contribute strong technical guidance to collaborative decisions.
+- Demonstrated ability to take a nebulous project, perform additional research, architect a solution, break the work into subtasks, and coordinate delivery quickly and efficiently among multiple engineers.
+- Demonstrated ability to diagnose, fix, and coordinate deployment of bug fixes -- and to do this extremely quickly in emergency situations.
+## Responsibilities & Expectations
+
+<!-- TODO: Add summary -->
+
+### Technical Skills
```

Comment:
You know what @akubetin - I just realized that from Engineer I on up, you've never focused too heavily on the lingo of our stack and I think that's a great move! I think this speaks to our intent to provide engineers across the stack opportunity for growth and clear benchmarks.